packages/bot-web-ui/src/components/dashboard/tour-guide.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { Text } from '@deriv/components';
+import RootStore from 'Stores/index';
+import { connect } from 'Stores/connect';
+
+type TTourGuide = {
+    label: string | boolean;
+    content: string[];
+    img?: string;
+    className: string;
+    dashboardTabIndex: number;
+    setActiveTab: (param: number) => void;
+};
+
+const TourGuide = ({ label, content, img, className, dashboardTabIndex, setActiveTab }: TTourGuide) => {
+    React.useEffect(() => {
+        setTimeout(() => {
+            setActiveTab(dashboardTabIndex);
+        }, 50);
+    }, [dashboardTabIndex]);
+    return (
+        <React.Fragment>
+            <div>
+                <Text as='h' line_height='l' weight='bold' className='intial_bot_tour'>
+                    {label}
+                </Text>
+            </div>
+
+            {img && (
+                <div className={className}>
+                    <img src={window.location.origin + img} />
```

Comment:
```suggestion
                    <img src={img} />
```
since there is a URLBase function which returns the relative path, this shouldn't be necessary right? 🤔

Reply:
done ✅